### PR TITLE
US129449 [AQ] Add Quiz Reliability Chart Card to Settings Page

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US129449](https://rally1.rallydev.com/#/?detail=/userstory/602529802561&fdp=true): [AQ] Add Quiz Reliability Chart Card to Settings Page

* Functional Testing 
  * [x] Browsers (Chrome, FF, Edge)
  * [x] Accessibility
  * [x] RTL
  * [x] Responsive
  * [x] Unit tests
* Security, devops, perf: N/A
* UX and docs
  * [x] Update SVG 
  * [x] Demo to Kevin
  
FYI @Brightspace/draco-vis

### Screenshot

image

### CODE REVIEW

File: test/visual-diff/quiz-item-analysis-view-skeleton.visual-diff.js [hunk 1]
```diff
@@ -10,7 +11,7 @@ const VisualDiff = require('@brightspace-ui/visual-diff');
 		let browser, page;
 
 		before(async() => {
-			browser = await puppeteer.launch();
+			browser = await puppeteer.launch(); 	// { headless: false } - useful for debugging
```
Comment (bemailloux) [inline test/visual-diff/quiz-item-analysis-view-skeleton.visual-diff.js:14]: Thanks!

File: test/visual-diff/helpers.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		if (target !== null) return target;
+		await new Promise(res => setTimeout(res, 5000));
+	}
+
+	return target;
+};
+
+/**
+ * @param {*} page - instance of puppeteer's page
+ * @param  {...any} selectors - array of valid selectors withing single shadow DOM
```
Comment (bemailloux) [inline test/visual-diff/helpers.js:23]: Generally selectors have type `String`, right?

File: test/visual-diff/helpers.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ * @returns an element that is rendered inside shadow DOM. It can handle nested shadow DOM's
+ */
+exports.queryShadowDom = async(page, ...selectors) => {
```
Comment (bemailloux) [inline test/visual-diff/helpers.js:26]: Is this to try and reduce the flakiness of the visual-diffs?

**Review by bemailloux — COMMENTED** [review]
LGTM! Waiting until it's not a draft PR for approval

File: test/visual-diff/helpers.js (hunk 3 above)
Comment (Vitalii-Misechko) [reply test/visual-diff/helpers.js:26]: That's the goal )

Comment (Vitalii-Misechko) [reply test/visual-diff/helpers.js:26]: Yes, it is. The idea is to wait on rendering of some element deep inside of DOM hierarchy and then take a snapshot. I hope it'll help to eliminate some kind of problems related to flakiness

**Review by bemailloux — APPROVED** [review]

**Review by NicholasHallman — APPROVED** [review]
LGTM!

**Comment by Vitalii-Misechko** [comment]
FYI @Brightspace/draco-vis

I've tried a few ways to get rid of flaky tests related to the hidden `d2l-overflow-group` element. I've discovered that chromium-browser reports that this element is visible (see `printElementVisibleProperties`), but screenshots still do not contain `d2l-overflow-group`. I assume that problem lays in the realm of OS (ubuntu) graphic subsystem interacting with chromium. For some reason, the screen that contains `d2l-overflow-group` is just skipped (throttling?) or is not shown at the moment of a screenshot. As result, we probably have only two options 1) restart the visual diff tests 2) change the timeout for a visual diff test in the hope we take a screenshot at the moment when the graphic subsystem shows all elements on the screen.

**Pull request merged**